src/Module.Server/ModuleData/dtv_data.xml

```diff
@@ -83,11 +87,11 @@
 
   <Round reward="320">
     <Wave>
-      <Group classDivisionId="crpg_dtv_samurai_warrior" count="1" />
+      <Group classDivisionId="crpg_dtv_samurai_warrior" count="0.8" />
```

Comment:
I don't think that wave is very hard so I think I would keep it that way

Reply:
done